doc/application_workflow/private_registry.md

```diff
@@ -0,0 +1,102 @@
+
+In this example, we are deploying an insecure registry with no authentication for simplicity.  We will also be targeting an ESXi environment.
+
+## Workflow
+
+In terminal #1: (local Docker)
+
+1. Open a terminal and make sure that it will use the local Docker daemon.  At a command prompt, issue  
+```
+    $> unset DOCKER_HOST
+```
+2. Install a VCH for a private registry using vic-machine
+3. Run Docker's registry on the first VCH
+4. Install a second VCH for running applications.  Make sure to specify --docker-insecure-registry.  This will make sure this second VIC can pull images from the insecure registry in the first VCH.
+5. At a terminal command prompt, using regular Docker, tag the images to be destined for the registry.
+6. Modify the docker systemd config file to allow pushing to an insecure registry
+7. Restart the docker daemon
+8. Push the image using the full tagged name (including host IP and port)
+
+In terminal #2: (VIC VCH)
+
+1. Open a terminal and make sure it is using the second VCH.  At a command promt, issue  
+```
+    $> export DOCKER_HOST=tcp://<VCH_IP>:<VCH_PORT>
+    $> export DOCKER_API_VERSION=1.23
+```
+2. Pull the image from the registry VCH
+
+### Example run
+
+terminal 1:
+```
+$> unset DOCKER_HOST
+$> ./vic-machine-linux create --target=192.168.218.207 --image-store=datastore1 --name=vic-registry --user=root --password=vagrant --compute-resource="/ha-datacenter/host/esxbox.localdomain/Resources" --bridge-network=vic-network --no-tls --volume-store=datastore1/registry:default --force
+
+...
+INFO[2016-10-08T17:31:06-07:00] Initialization of appliance successful       
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] vic-admin portal:                            
+INFO[2016-10-08T17:31:06-07:00] http://192.168.218.138:2378                  
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] Docker environment variables:                
+INFO[2016-10-08T17:31:06-07:00]   DOCKER_HOST=192.168.218.138:2375           
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] Connect to docker:                           
+INFO[2016-10-08T17:31:06-07:00] docker -H 192.168.218.138:2375 info          
+INFO[2016-10-08T17:31:06-07:00] Installer completed successfully             
+
+$> DOCKER_HOST=tcp://192.168.218.138:2375 DOCKER_API_VERSION=1.23 docker run -d -p 5000:5000 --name registry registry:2
+
+$> ./vic-machine-linux create --target=192.168.218.207 --image-store=datastore1 --name=vic-app --user=root --password=vagrant --compute-resource="/ha-datacenter/host/esxbox.localdomain/Resources" --bridge-network=vic-network --no-tls --volume-store=datastore1/vic-app:default --force --docker-insecure-registry 192.168.218.138
```

Comment:
Looks like it: https://github.com/vmware/vic/blob/master/cmd/vic-machine/create/create.go#L383